.openpublishing.redirection.json

```diff
@@ -908,6 +908,10 @@
             "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md",
             "redirect_url": "/dotnet/visual-basic/getting-started/breaking-changes-in-visual-studio"
         },
+        {
+            "source_path": "docs/visual-basic/reference/vb6-support.md",
+            "redirect_url": "/visualstudio/vb6/vb6-support"
+        },
```

Comment:
I think you want to add the "redirect_document_id": true here since they're the same topic

Reply:
@mairaw, I've added it. I'll merge when the build completes.